tests/services/test_barcode_cobranca.py

```diff
@@ -58,3 +60,81 @@ def test_code_line_to_barcode(self):
                 result = BarcodeCobrancaService().code_line_to_barcode(code_line)
 
                 self.assertEqual(result, barcode)
+
+    def test_calculate_due_date_1(self):
+        """
+        Dado:
+            - Um Fator de Vencimento fv = 1009
+        Quando:
+            - É dia 12/02/2025
+            - BarcodeCobrancaService().calculate_due_date(fv)
+        Então:
+            - Deve ser retonado 03/03/2025
+        """
+
+        fv = "1009"
+
+        date = datetime.strptime("2025-02-12", "%Y-%m-%d")
+
+        with freeze_time(date):
+            result = BarcodeCobrancaService().calculate_due_date(fv)
+
+        self.assertEqual(str(result), "2025-03-03 00:00:00")
```

Comment:
O resultado deveria ser um date, e não um datetime!

```suggestion
        self.assertEqual(str(result), "2025-03-03")
```